### ISSUE TITLE
both "Save" and "Save as" work similarly even the label file exists!

### ISSUE THREAD

**Issue by ma-mehralian** [tzutalin/labelImg#328]
- Win 7
- PyQt 5
The "Save" button opens save dialogue even if the label file (i.e. *.xml) exists before.
However, it is expected to overwrite the pre-exist label file which labels were read

**Comment by vdalv** [comment]
Yeah, the save button should overwrite the pre-existing label file. I'm not having this issue running the latest code on Win 10.

If you are running from source, please make sure to pull the latest code.
If you are running from binary, please make sure to use the latest binary.

If that doesn't solve the issue (or you're already running the latest code/binary), please post the steps we can take to reproduce this issue.

Thanks,

-Vlad

**Comment by ma-mehralian** [comment]
Simply run labelImg, choose "Open Dir", select pre-labeled image from "File List" and change a box's label from sidebar then select "Save".

I test it on both latest binary (1.7.0 [sys v: 2.7.8]) and latest code(1.7.0 [sys v: 3.6.4]). 
It should be note that after the first saving, the "Save" button works properly for other modification of current image

Thanks

**Comment by ma-mehralian** [comment]
In "Auto Saving" mode, labelImg asks for destination directory. After switch to "Auto Saving" mode the label files overwrite automatically.

My problem was solved with this method, but it seems that there is still a problem with the program.

Thanks

**Comment by vdalv** [comment]
Thanks for the clarification, I'm not too familiar with the Auto Saving mode, so I'll have to look into this at a later time.

I believe this issue can also be resolved in your case by setting the save directory by clicking the "Change Save Dir"

**Comment by jinfagang** [comment]
@ma-mehralian How to enable auto-save mode?
